### PR TITLE
Airport challenge - Rich

### PR DESCRIPTION
**Pull request by RJ21BL**
I done four user stories.
Struggled with capacity of airport, feedback required.

### CODE REVIEW

File: lib/airport.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      return true
+    end
+  end
+
+  def take_off(plane)
+    if @plane == nil
+      return true
+    end
+  end
+
```
Comment (alangoodier) [inline lib/airport.rb:20]: My only piece of feedback would be to move this section below 'def initialize' to the top underneath the class heading. Then you can see these attributes first.

**Review by alangoodier — COMMENTED** [review]
It is very heard to pick out any issues - given that you got further than me in the challenge! But your code is easy to read and it makes sense. There aren't any odd method or variable names that I can see. Overall a great effort and it's easy to read.

**Review by leoht — COMMENTED** [review]
Nice work, it'll be good to revisit this challenge with the new things you'll have learned, at the end of the week if you want (or at any point later). I've left some comments - a good way of knowing what to write in the methods is to first try to write on a piece of paper what we want to do. For example: 

-> What does it mean for a plane to land at the airport?
-> For our program, it means setting the `@plane` attribute of the airport

So to check the plane is landed, what we're doing is:
-> checking whether the `@plane` attribute is set or not

Now try to think about what it means for a plane to take-off - what will we need to do in respect of the `@plane` attribute?

File: lib/airport.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  attr_accessor :planes, :capacity
+  DEFAULT_CAPACITY = 10
```
Comment (leoht) [inline lib/airport.rb:22]: Nice use of constants! :)

**Pull request closed without merging**